src/crypto.h

```diff
@@ -209,6 +209,9 @@ int
 _libssh2_ecdsa_curve_type_from_name(const char *name,
                                     libssh2_curve_type *out_type);
 
+void
+_libssh2_ecdsa_free(libssh2_ecdsa_ctx* ctx);
```

Comment:
This line is causing some of the CI failures. It can be safely deleted.
Remapping also remaps the declaration, so this one comes out as duplicate.

Reply:
Added 

```
void
_libssh2_wincng_ecdsa_free(libssh2_ecdsa_ctx* ctx);
```

in `wincng.h` instead, that seems to fix it.